### DISCUSSION TITLE
Figma Design Toolkit v17

### DISCUSSION THREAD

**Discussion by d-m-s** [vmware-clarity/ng-clarity#1303] [Q&A]
Is there an updated Figma Design Lib available?

The v17 release announcement mentioned an update to the Design Toolkit, but the ones listed at https://www.figma.com/@vmware are about 3 years old.

https://clarity.design/

<img width="698" alt="2024-03-05_11h05_51" src="https://github.com/vmware-clarity/ng-clarity/assets/311619/f741a7ba-0319-414b-9829-eccfde773f2c">

**Comment by TekSiDoT** [comment]
@kristinaross any chance you could take a look at this?

**Reply by d-m-s** [reply]
@kevinbuhmann can you give us an update on this, or probably redirect the question?

**Reply by kevinbuhmann** [reply]
Unfortunately, the v17 Figma Toolkit is not available publicly. I have flagged this internally. The language on the website should be clarified or removed soon.

**Reply by TekSiDoT** [reply]
Are there any plans to release Figma Toolkits again in the future? Working with Design Tokens doesn't make any sense otherwise.

**Reply by d-m-s** [reply]
Thanks Kevin for you feedback!

Some of us accompany and support Clarity for a long time now ...
the beginning was inspiring - i'd hope for the project to see a turn in this direction once again.

https://medium.com/claritydesignsystem/one-year-of-clarity-4e87ee66c3a3

### **Embrace open source**

This was a tough one for us at VMware but if you’re going open source, go fully open source. Clarity does not maintain any internal issue tracking software, we do not release internal builds of Clarity, and we do not prioritize issues outside of the community’s eyes.
 
**All of Clarity is open source. This is important.** It’s been important in making sure our community can build the trust in us and our process but it has also allowed us to maintain the integrity of Clarity and our technical architecture. Being open source allowed to build software we’re proud of sharing externally and we’re able to actually share in a way a design system should: generically.

**Reply by d-m-s** [reply]
@kristinaross @kevinbuhmann @pdaigle @Jinnie

**Any updates or feedback from Broadcom/VMware?**

![2024-05-24_14h19_45](https://github.com/vmware-clarity/ng-clarity/assets/311619/33ac661e-1e8e-4ada-8dee-23a94ef42aa4)

**Reply by kevinbuhmann** [reply]
We updated the website in April to note that the v17 Figma toolkit is "coming soon". I don't know anything more specific.

**Reply by d-m-s** [reply]
Thanks @kevinbuhmann, regarding giving feedback to the community it seems like you're the last man standing in Clarity's team!
Thanks at least for your engagement, but please escalate this issue further up to management!

**The current status quo of community-communication and support is not sustainable!**

@squidjam and @TekSiDoT already pointed it out spot on [here](https://github.com/vmware-clarity/core/discussions/284):

**Quote 1:**
> As an outside observer, Clarity Design System (or at least the Open Source part visible from the outside), has been in a sorry state for the past 2-3 years.
> 
> - There is no roadmap for new features
> - Features that have been promised and delayed again and again in the name of accessibility are still nowhere in sight
> - The Core/Angular split has led to two understaffed projects
> - Important Design related work has not been updated (The Figma lib saw its last update 3 years ago)
> - References to internal channels and emails have increased.

**Quote 2:**
> Thanks for your response, but you will understand how confusing it is that issues and discussions go without responses of any kind for years.
> I really hope communications starts to improve and that `core` starts getting the same kind of love as `ng-clarity`.
